zstack_dashboard/static/templates/vm/createVm.html

```diff
@@ -2,7 +2,16 @@
     <div class="row">
         <div class="col-sm-15">
             <div class="tab-content">
-                <div id="createVmInstanceInfo" class="tab-pane active">
+                <div ng-if="isBatchMode()" id="createVmInstanceCount" class="tab-pane active">
```

Comment:
This small page will let following pages inherent the same offset to top and part of page will out of view. Maybe move count input control to next page is better.